go.mod

```diff
@@ -4,6 +4,7 @@ require (
 	github.com/google/go-cmp v0.2.0
 	github.com/pmezard/go-difflib v1.0.0
 	golang.org/x/crypto v0.0.0-20190422183909-d864b10871cd // indirect
+	golang.org/x/exp v0.0.0-20190510132918-efd6b22b2522 // indirect
```

Comment:
Could you run `go mod tidy` on this as you've only format one file and didn't really introduce new dependency.

Reply:
I believe these are from `go install -mod=readonly golang.org/x/exp/cmd/apidiff`. Maybe it would be better to remove the `-mod=readonly` from that command instead of adding the dependencies to the `go.mod`?

Reply:
OK, I ran `go mod tidy` and updated the script that installs `apidiff` to `cd` to a tempdir first so that it doesn't try to update the project `go.mod`.